charts/ccd-docker-user-profile-importer/values.yaml

```diff
@@ -0,0 +1,19 @@
+importer:
```

Comment:
can we change these to root level as this is not a sub chart ?

Reply:
fixed